launch/start.launch

```diff
@@ -123,7 +123,9 @@
 	<node pkg="hero_bridge" type="tf_ghost.py" name="tf_ghost"/>
 	<node pkg="hero_bridge" type="head_controller.py" name="head_bridge"/>
 	<node pkg="hero_bridge" type="manipulation_bridge.py" name="manipulation_bridge"/>
-	<node pkg="hero_bridge" type="handover_detection.py" name="handoverdetector"/>
+	<node pkg="hero_bridge" type="handover_detection.py" name="handoverdetector"/>>
```

Comment:
```suggestion
	<node pkg="hero_bridge" type="handover_detection.py" name="handoverdetector_bridge"/>>
```

Reply:
That has nothing to do with this PR, right?